### DISCUSSION TITLE
[Client idea] simulate keyboard input

### DISCUSSION THREAD

**Discussion by kmahyyg** [dani-garcia/vaultwarden#1949] [Ideas]
For some native client, click copy and paste is working. But if the client disabled copy and paste password, such as an old-fashioned custom ActiveX plugin or some custom widget in UI, I need to type char by char.

Is this possible to simulate keyboard input? just like 1password, click a textbox, then find corresponding item, then click fill even it is a native software.

And also, for those situations you must input the password char by char, is this possible to zoom in the password string, which would be much friendly for users such as chars like `i` (lower-case india) or `I` (upper-case india) or `l` (lower-case lima).

**Comment by Masgalor** [comment]
You may be not ware, but vaultwarden only covers the backend api server implementation.

I think it would be better to present your idea to the bitwarden project, since that is where the actual clients are developed.
https://github.com/bitwarden
